common/shlibs

```diff
@@ -986,6 +986,7 @@ libev.so.4 libev-4.04_1
 libgvnc-1.0.so.0 gtk-vnc-0.4.4_1
 libgtk-vnc-2.0.so.0 gtk-vnc-0.4.4_1
 libyaml-0.so.2 libyaml-0.1.4_1
+libuchardet.so.0 uchardet-0.0.6_1
```

Comment:
problably future user of this library ?

Reply:
https://github.com/Johnnynator/void-packages-1/blob/deepin/srcpkgs/dde-file-manager/template#L10